rtc/configuration.md

```diff
@@ -359,14 +433,26 @@ The same as [`rtc_client_connected`](#rtc_client_connected)
 
 ### Example using client connect and disconnect events with custom user details
 
+While all fields are provided to both `rtc_client_connected` and `rtc_client_disconnected` functions, this example shows only relevant fields need to be used.
+
 ```js
+const connectedUsers = {}
 tinymce.init({
   selector: 'textarea', // change this value according to your HTML
   plugins: 'rtc',
   rtc_user_details_provider: ({userId}) => Promise.resolve({ fullName: "John Doe", jobTitle: "Engineer" }),
-  rtc_client_connected: ({userDetails: {fullName, jobTitle}, caretNumber}) =>
-    console.log(`${jobTitle} ${fullName} connected with caret number ${caretNumber}`)
-  rtc_client_disconnected: ({userId, clientId, caretNumber, clientInfo}) =>
-    console.log('Disconnected', userId, clientId, caretNumber, clientInfo)
+  rtc_client_connected: ({userDetails: {fullName, jobTitle}, userId, caretNumber, clientId, clientInfo}) => {
```

Comment:
I realise this part is unchanged, but with the amount of info in here now it might be worth only doing one level of de-structuring?

Reply:
Yes, this is just me sneaking in some developer education with code golf. I also think it looks better than later doing 
```
console.log(`${userDetails.jobTitle} ${userDetails.fullName}....
```